egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -37,49 +37,19 @@ def GetArgs():
     parser = argparse.ArgumentParser(description="""
     Trains RNN and DNN acoustic models using the 'chain' objective function.
     """,
-    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    conflict_handler = 'resolve')
```

Comment:
If there are any values that may be overridden, then comment next to the original definition that it may be overridden in the command-line scripts, and comment next to the overriding one that it overrides an existing declaration.  But I think it's better if you remove this option and do not use this feature.  BTW, this does not happen in the C++ binaries-- they would crash at runtime if you were to override an option.